articles/active-directory-b2c/openid-connect.md

```diff
@@ -64,25 +64,25 @@ At this point, the user is asked to complete the workflow. The user might have t
 
 After the user completes the user flow, a response is returned to your application at the indicated `redirect_uri` parameter, by using the method that's specified in the `response_mode` parameter. The response is the same for each of the preceding cases, independent of the user flow.
 
-A successful response using `response_mode=fragment` would look like:
+A successful response using `response_mode=fragment` and `response_type=code+token` would look like:
 
 ```http
-GET https://aadb2cplayground.azurewebsites.net/#
-id_token=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6Ik5HVEZ2ZEstZnl0aEV1Q...
-&code=AwABAAAAvPM1KaPlrEqdFSBzjqfTGBCmLdgfSTLEMPGYuNHSUYBrq...
+GET https://jwt.ms/#
+access_token=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dCI6Ik5HVEZ2ZEstZnl0aEV1Q...
+&code=eyJraWQiOiJBQjk4dkdFRHY2Nl9ScmJEajJmclFFMUFOM1hSX3k4ZVU3X3V5NDBjN...
 &state=arbitrary_data_you_can_receive_in_the_response
```

Comment:
On the response, only update the GET URL to https://jwt.ms/#, leave everything else as is. We won't receive an access token from `/authorize` endpoint using code+id_token response type.